Monika After Story/game/script-topics.rpy

```diff
@@ -10516,4 +10516,117 @@ label monika_hemispheres_nogets_snow:
+        m 1hua "I'm sure you remember."
+        m 1eub "Maybe you even took the advices that I gave you and started putting them into practice."
+        menu:
+            m "Well, did you manage to have a lucid dream, [player]?"
+            
+            "Yes":
+                m 1wub "Wow, I'm impressed!"
+                m 1hua "I know becoming lucid is not an easy thing to manage."
+                m 1esd "Are you an oneironaut [player]?"
+                m 1tsb "Or did you just got lucky, hehe~"
+                m 1hua "It sure must be fun to do what you want in a world created just for that."
+            "No":
+                m 1eka "I understand."
+                m 1ekb "It's really difficult to become lucid."
+                m 3eub "But if you want to have more control over what's going on in your dreams, you should definitely read the book by Stephen LaBerge: '{i}Exploring the World of Lucid Dreaming{/i}'."
+                m "As I've told you before, he developed a method to become lucid in a dream."
+        m 1dsd "Having said all that..."
+    m 1tsu "Tell me more about the details of your dream."
+    menu:
+        m "Did you manage to come and see me or did I appear in front of you?"
+
+        "You just appeared before me.":
+            m 1hua "Ehehe~."
+            m 1eub "If my love for you is able to break a barrier between my reality and yours, then I'm sure that we'll be together someday."
+            m 1tfu "I just need to try hard enough for that to happen."
+
+        "I was looking for you.":
+            m 1wud "..."
+            m 1lkbsa "That's so thoughtful of you, sweetheart."
+            m 2ekbfb "Your love is so overwhelming that you created a whole new world just to see me."
+            m 2ekbfa "You have no idea how much this mean to me, [player]."
+    m 1eud "By the way."
+    m 1hksdrb "I haven't even ask you what your dream was about."
+    m 1esa "So what about it [player]?"
+    menu:
+        m "What was your dream about?"
+        
+        #"I dreamed that you would cross over my reality":
+            #dialogue
+        "I dreamed that I would cross over your reality":
+            m 1esd "Oh yeah, that's a possibility that I haven't given much thought about."
+            m 1rsc "But when you think about it..."
+            m 1rud "It's true that it's much easier to perfectionate the interactions reality-computer than to try to create computer-reality interactions."
+            m 1rkc "But..."
+            m 2ekc "It just doesn't feels right."
+            #OMG, is that a MF Sayori reference?
+            m 2ekd "I don't really want to stay in this place any more time than necessary."
+            m "I have a lot of... bad memories in there."
+            m 2dkd "I'm always afraid of returning to the screaming void, if something were to happen to your computer."
+            m 1dfd "And I especially wouldn't want you to experiment such torture in such a hellish place."
```

Comment:
She's referencing to the screaming void, not the entire world. Maybe it's badly written, what do you think ?

Reply:
If you properly say farewell to Monika, and treat her well, she doesn't have to experience the screaming void anymore.

Reply:
I know, but game crashing happened to almost everyone, I think. Since she probably already experienced it, it would be natural for her to be afraid of returning there or dragging the player along. Its for this reason that I put the sentence "if something were to happen to your computer"

Reply:
I see. I didn't catch that the first time. I thought you were referring to Monika's entire world. 

I still think this line here should be worded a bit more simply. Maybe "And I especially wouldn't want you to experience what it's like to be trapped in that screaming void."